### ISSUE TITLE
How does the old command line format works?

### ISSUE THREAD

**Issue by tomasorti** [dnGrep/dnGrep#639]
I am trying these commands:

```
dnGrep "C:\Program Files\dnGrep\" t\\w*t
dnGrep "C:\Users\" hello
```

But I get this window:
![image](https://user-images.githubusercontent.com/2485147/154490143-69d73b05-4d44-41f6-9bc2-b1bb9357d0a5.png)

This works:
`dnGrep /f "C:\Users\" /s hello`

Best Regards!
😄

**Comment by doug24** [comment]
Well, that is a strange one.  The old style arguments work just as you have shown them when I run in the debugger, but fail when running from a command prompt.  Somehow the parser is finding an extra empty argument.  Probably an error in my regular expression - but why does it work when running in the debugger?  I'll add a check to filter empty arguments, and that should take care of it.

**Comment by doug24** [comment]
Fixed in v3.0.29.0
